platform/lang-impl/src/com/intellij/codeInsight/documentation/DocumentationComponent.java

```diff
@@ -776,14 +779,73 @@ private void setDataInternal(@Nullable SmartPsiElementPointer<PsiElement> elemen
                                @Nullable String ref) {
     myIsEmpty = false;
     if (myManager == null) return;
-
-    myText = text;
     setElement(element);
+    
+    text = fixImageReferences(text);
+    myText = text;
     myDecoratedText = decorate(text);
-
     showHint(viewRect, ref);
   }
 
+  /**
+   * Fix the <img> elements references by adding references to local files if necessary
+   *
+   * @param text: an html string is expected. If it is not an html
+   *              then it returns the value passed by parameter
+   */
+  private String fixImageReferences(String text) {
+    if (text == null || text.isEmpty() || !BasicHTML.isHTMLString(text)) {
+      return text;
+    }
+    text = text.replace("<br>", "<br/>");
+    String basePath = this.myManager.myProject.getBasePath();
+    try {
+      org.jdom.Element dom = JDOMUtil.load(text);
```

Comment:
Sorry, this approach won't work: `text`  is an HTML document with all funny things like unclosed tags and unquoted attribute values, which makes it unreadable by XML parsers.
Also, I don't think that it's a proper place to fix Python specific problem, maybe `com.jetbrains.python.documentation.PythonDocumentationProvider` would be a better place.

Reply:
@vlasovskikh